tools/validate_spec.py

```diff
@@ -226,28 +398,10 @@ def __exit__(
             self.file.write("\n]")
             self.file.close()
 
-        if exception_value is None and self.classified_incorrectly != 0:
-            raise ValidationError(
-                f"{self.classified_incorrectly} messages were classified incorrectly"
-            )
-
     def write_result(self, validation_result: ValidationResult) -> None:
-        self.__write_console_output(validation_result)
+        validation_result.print_console_output()
         self.__write_json_output(validation_result)
```

Comment:
Why is `validation_result.print_console_output` called inside a method of `OutputWriter`?

Reply:
Moved the call into `validate` in e020204d